haystack/document_stores/in_memory/document_store.py

```diff
@@ -68,6 +68,7 @@ def __init__(  # pylint: disable=too-many-positional-arguments
         embedding_similarity_function: Literal["dot_product", "cosine"] = "dot_product",
         index: Optional[str] = None,
         async_executor: Optional[ThreadPoolExecutor] = None,
+        return_embedding: bool = True,
```

Comment:
Let's add a docstring for `return_embedidng`

Something like `:param return_embedding: Whether to return the embedding of the retrieved Documents.`